src/expr/src/scalar/func.rs

```diff
@@ -2053,6 +2053,7 @@ pub enum BinaryFunc {
     Lte,
     Gt,
     Gte,
+    IsILikePatternMatch,
     IsLikePatternMatch,
```

Comment:
I think we should make this work like `IsRegexpMatch`, where you stick a `case_insensitive: bool` inside the variant!

Reply:
I just caught this as I pushed! Working on it now.